writing_checks/README.md

```diff
@@ -468,6 +468,38 @@ whereas if it run on a system with ALPS it will be translated to
 ddt --offline aprun ...
 ```
 
+## Custom Launchers
+
+ReFrame permits the simple addition of custom scheduler launchers. A launcher that invokes `mpirun` can for example be added as follows:
```

Comment:
Put each sentence in a separate line. This way we avoid conflicts in the future, or we can resolve them faster if they occur.